src/graphs/lca.rs

```diff
@@ -53,7 +55,7 @@ impl LCA {
             }
         }
         let d_with_order: Vec<(usize, usize)> = order.iter().map(|&u| (d[u], u)).collect();
-        let rmq = RMQ::new(&d_with_order, std::cmp::min);
+        let rmq = RMQ::<(usize, usize), OpType>::new(&d_with_order, std::cmp::min);
```

Comment:
although I still couldn't figure out how to get this to work the other way: capturing depth array